e2e/cypress/support/api/user.js

```diff
@@ -203,3 +203,16 @@ Cypress.Commands.add('apiGetUsersNotInTeam', ({teamId, page = 0, perPage = 60} =
         cy.wrap({users: response.body});
     });
 });
+
+Cypress.Commands.add('apiDeactivateUser', (userId) => {
```

Comment:
Please let me know if it's the correct place to add a command.

Reply:
That's perfect. Thanks for adding this command 💯

Reply:
Agreed. Useful command 👍